Nos/Controller/FeedController.swift

```diff
@@ -0,0 +1,244 @@
+import Combine
+import CoreData
+import Dependencies
+import SwiftUI
+
+/// The source to be used for a feed of notes.
+enum FeedSource: RawRepresentable, Hashable, Equatable {
+    case following
+    case relay(String, String?)
+    case list(String, String?)
```

Comment:
can you used named tuple arguments here? That's a thing right? If not then at least some docs.
```suggestion
    case list(name: String, description: String?)
```